packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -627,11 +627,16 @@ export class ConnectionFlow {
       this.account = await prepareTriggerAccount(this.client, this.trigger)
     }
     if (konnectorPolicy.onLaunch) {
-      konnectorPolicy.onLaunch({
-        konnector: this.konnector,
-        trigger: this.trigger,
-        account: this.account
-      })
+      konnectorPolicy
+        .onLaunch({
+          konnector: this.konnector,
+          trigger: this.trigger,
+          account: this.account
+        })
+        .then(() => this.setState({ status: IDLE }))
+        .catch(err =>
+          logger.error(`Error while launching policy : ${err.message}`)
+        )
```

Comment:
Is it on purpose to not block the process with an `await`? Because even if an error is thrown, we will subscribe to the realtime and so on.  If this is done for purpose, than can you explain why in the commit?

Reply:
This was on purpose. I added explanations about it in the commit